rollup.config.js

```diff
@@ -18,17 +19,36 @@ const banner = `/*!
 `;
 
 module.exports = () => ({
-  input: './src/api.mjs',
+  input: './src/api.mts',
   plugins: [
     json({ compact: true }),
     commonjs(),
+    mtsResolver(),
     nodeResolve(),
     babel({
       babelHelpers: 'bundled',
       exclude: 'node_modules/**',
+      presets: [[
+        '@babel/preset-env',
```

Comment:
i don't think we need to do babel envs and such here. if someone wants to run this on old safari they can babel it themselves. one of the goals of this package is that the built output is as similar to the source as possible so that source map errors are as small as possible so that you can easily use js debuggers to step through it.

Reply:
Oh, I thought engine262 needs to support some old browser (because I see it contains a transform for `?.` syntax).

Is it ok to use the latest syntax? (that means class initialization block comes in)

Reply:
i guess the main requirement is that it runs in node, which usually lags slightly behind browsers, so as long as it runs alright in node its usually fine.